### PR TITLE
Fix issue with sample page when screenshot tool is enabled

### PR DESCRIPTION
**Pull request by duffh**
# Description

Fixed an issue that resulted in the source code and description pages showing behind the sample container when the screenshot tool is enabled. 

## Type of change

- Bug fix

## Platforms tested on

- [x] MAUI WinUI
- [x] MAUI Android
- [x] MAUI iOS
- [x] MAUI MacCatalyst

## Checklist

- [x] Self-review of changes
- [x] All changes work as expected on all affected platforms
- [x] There are no warnings related to changes
- [x] Code is commented and follows .NET conventions and standards
- [x] Codemaid and XAML styler extensions have been run on every changed file
- [x] No unrelated changes have been made to any other code or project files

### CODE REVIEW

**Review by amweis226 — CHANGES_REQUESTED** [review]
The actual changes work great, on resolving the metadata checks (looks like a lot of jpg files come up missing at first glance) I'll approve

**Comment by duffh** [comment]
> The actual changes work great, on resolving the metadata checks (looks like a lot of jpg files come up missing at first glance) I'll approve

The checks were failing in error, a rerun passed all checks. 

I'll merge once you have approved.

**Pull request merged**
